src/coreclr/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -46,7 +46,8 @@
     <!-- Override InformationalVersion during servicing as it's returned via public api. -->
     <InformationalVersion Condition="'$(PreReleaseVersionLabel)' == 'servicing'">$(ProductVersion)</InformationalVersion>
     <InformationalVersion Condition="'$(StabilizePackageVersion)' == 'true'">$(ProductVersion)</InformationalVersion>
-    <NoWarn>$(NoWarn),0419,0649,CA2249,CA1830</NoWarn>
+    <!-- AD0001 : https://github.com/dotnet/runtime/issues/90356 -->
+    <NoWarn>$(NoWarn),0419,0649,CA2249,CA1830;AD0001</NoWarn>
```

Comment:
Should such a NoWarn be centralized in https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems?

Reply:
I considered it but the generator was referenced in each project and it made more sense to put this in the same spot the generator was referenced.